cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DistributedPreviewManager.java

```diff
@@ -244,6 +249,13 @@ public void run() {
             }
           }
 
+          // Overwrite JVM options if specified
+          Set<String> runnables = Stream.of(PreviewRunnerTwillRunnable.class.getSimpleName(),
+                                            ArtifactLocalizerTwillRunnable.class.getSimpleName())
+            .collect(Collectors.toSet());
+          Map<String, String> jvmOverrideConfig = cConf.getPropsWithPrefix(Constants.AppFabric.PROGRAM_JVM_OPTS_PREFIX);
```

Comment:
I believe this is left behind after your refactoring.

Reply:
Fixed